packages/kbn-generate-csv-types/index.ts

```diff
@@ -24,4 +24,5 @@ export interface CsvConfig {
     duration: string;
     size: number;
   };
+  maxConcurrentShardRequests: number;
```

Comment:
This change conflicts with #162845, which has been on hold for a long time so I'd like to get that in first.

The resolution will be to import `ReportingConfigType` from `@kbn/reporting-server`

```diff
-import { CsvConfig } from '@kbn/generate-csv-types';
+import { ReportingConfigType } from '@kbn/reporting-server';

-CsvConfg
+ReportingConfigType['csv']
```